packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.test.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import {render, screen, waitFor} from '@testing-library/react'
+import {StorefrontPreview, detectStorefrontPreview} from './index'
+
+jest.mock('./index', () => {
+    const origin = require('./index')
+    return {
+        ...origin,
+        detectStorefrontPreview: jest.fn()
+    }
+})
+describe('Storefront Preview Component', function () {
+    const OLD_ENV = process.env
+    beforeEach(() => {
+        process.env = {...OLD_ENV}
+    })
+    afterEach(() => {
+        process.env = OLD_ENV
+    })
+    test.only('renders script tag when enabled is on', () => {
+        process.env.STOREFRONT_PREVIEW = true
```

Comment:
```suggestion
        process.env.STOREFRONT_PREVIEW = 'true'
```
All `process.env` values must be strings.